comps/comps-katello-client-fedora27.xml

```diff
@@ -19,7 +19,8 @@
       <packagereq type="default">katello-host-tools</packagereq>
       <packagereq type="default">python3-gofer</packagereq>
       <packagereq type="default">python3-gofer-proton</packagereq>
-      <packagereq type="default">python-gofer</packagereq>
+      <packagereq type="default">python2-gofer</packagereq>
+      <packagereq type="default">python2-gofer-proton</packagereq>
```

Comment:
Don't think we need python2-gofer-proton from the agent side  - we want to be on PY3

Reply:
Why do we need python2 gofer then?

Reply:
It's a quirk of the gofer.rpm packaging. It will bring python2-gofer whether you like it or not - doesn't seem to matter / break anything

Reply:
@pcreech is this something we can effect?

Reply:
So, by way of fedora packaging rules, the 'gofer' rpm is depending on 'python-gofer'.  This 'python-gofer' will always resolve to the system default python major version package (python2-gofer on f27)

My main question is, do you _need_ the 'gofer' package itself? the main things of concern in that package are (i think) /etc/gofer/*, /usr/bin/goferd and /usr/lib/systemd/system/goferd.service

If you don't need or create otherwise any of that stuff, you could _probably_ get away with just depending on the python3-* libraries only and not pulling in the 'gofer' package, reducing the amount of things you pull in.

if you do depend on goferd for katello-agent, then i think we won't be able to prevent pulling in the python2 stuff b/c goferd is set up to run under the system's default python, and will probably require fixes in gofer upstream.

Reply:
I'd say we definitely need those pieces